src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { films } from './../../MockedData'
+import { isEmpty, isNil } from 'ramda';
+
+const ErrorBoundary  = ({ children }) => {
```

Comment:
According to the task you need to use [ErrorBoundary component](https://reactjs.org/docs/error-boundaries.html)

Reply:
https://github.com/Azaliyaa/netflix-roulette/pull/1/commits/b8d79aed75d8b64298baffabf18ec52e6404b1dc